packages/trader/src/sass/app/modules/reports.scss

```diff
@@ -21,7 +21,7 @@ $side-padding: 1.2em;
             display: flex;
             width: 100%;
             @include desktop {
-                max-width: 36rem;
+                max-width: 50rem;
```

Comment:
Won't this break the current styles for anything?

Reply:
as per my observation it doesn't break anything